BooleanDemo.java

```diff
@@ -8,6 +8,7 @@ public static void main(String[] args) {
 		b= false;
 		System.out.println("new b value is - "+b);
 		System.out.println("10>9 is "+ (10>9));
+		System.out.println("Test Sonar");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)